lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -528,17 +529,23 @@ data ByronWalletPutPassphraseData = ByronWalletPutPassphraseData
     , newPassphrase :: !(ApiT (Passphrase "raw"))
     } deriving (Eq, Generic, Show)
 
+newtype ApiTransactionTTL = ApiTransactionTTL
+    { seconds :: NominalDiffTime
+    } deriving (Eq, Generic, Show)
+
 data PostTransactionData (n :: NetworkDiscriminant) = PostTransactionData
     { payments :: !(NonEmpty (AddressAmount (ApiT Address, Proxy n)))
     , passphrase :: !(ApiT (Passphrase "lenient"))
     , withdrawal :: !(Maybe ApiWithdrawalPostData)
     , metadata :: !(Maybe (ApiT TxMetadata))
+    , ttl :: !(Maybe ApiTransactionTTL)
```

Comment:
shall we call this: `timeToLive` for the sake of self-documentation. TTL is common enough, but I'd rather avoid acronym in the API.

Reply:
No problem.